src/panels/lovelace/editor/card-editor/hui-edit-card.ts

```diff
@@ -86,6 +87,22 @@ export class HuiEditCard extends LitElement {
     return this.shadowRoot!.querySelector("hui-card-preview")!;
   }
 
+  private get _cardJSON(): LovelaceCardConfig | undefined {
+    return this._cardConfig;
+  }
+
+  private set _cardJSON(config: LovelaceCardConfig | undefined) {
```

Comment:
What's the benefit of using a getter/setter instead of just a property?

Reply:
... umm... It was 2 AM...